File: api/v2010/member/fetch-example-1/fetch-example-1.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account.Queue;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var member = MemberResource.Fetch(
            pathCallSid: "CA386025c9bf5d6052a1d1ea42b4d16662",
            pathQueueSid: "QU5ef8732a3c49700934481addd5ce1659"
        );

        Console.WriteLine(member.CallSid);
    }
}

File: api/v2010/incoming_phone_number/fetch-example-1/fetch-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

incoming_phone_number = @client
  .incoming_phone_numbers('PN2a0747eba6abf96b7e3c3ff0b4530f6e')
  .fetch

puts incoming_phone_number.account_sid
